www/src/components/sidebar/sidebar.js

```diff
@@ -235,7 +235,7 @@ class SidebarBody extends Component {
                 key={index}
                 level={0}
                 location={location}
-                onLinkClick={closeSidebar}
```

Comment:
Ah, `closeSidebar`! That of course made a lot more sense in the context of Gatsby v1. All the sidebar stuff started as #6076 against `www` on v1 (and mostly borrowing the reactjs.org sidebar implementation 🙏), where the sidebars were part of the `layout`, and never got removed from the DOM when switching `location`. 

IMO `closeSidebar` does still make sense for Gatsby v2 `www` for mobile and the anchor links in the Tutorial (https://next.gatsbyjs.org/tutorial/part-zero/#overview-of-core-technologies)…having a hard time putting thoughts into words, long day…lemme record a video! 😄 

OK, here's the one good thing `closeSidebar` is still doing and IMO is worth preserving:
https://www.dropbox.com/s/yj6dl1wx788vpwm/closeSidebar.mov?dl=0